app/views/alchemy/admin/elements/_element.html.erb

```diff
@@ -14,7 +14,7 @@
       <div id="element_<%= element.id %>_errors" class="element_errors"></div>
 
       <div id="element_<%= element.id %>_content" class="element-content-editors">
-        <%== render_editor(element) %>
+        <%= render_editor(element) %>
```

Comment:
Why are you removing this here? Does it work without passing the `html_safe` check? I currently can't say if `render_editor` returns a HTML safe string.